wavefront-hpa-adapter/values.yaml

```diff
@@ -0,0 +1,28 @@
+image:
+  repository: wavefronthq/wavefront-hpa-adapter
+  tag: 0.9.3
+  pullPolicy: IfNotPresent
+
+apiserver:
+  # Wavefront url of the form https://INSTANCE.wavefront.com
+  # wavefrontURL: https://INSTANCE.wavefront.com
+
+  # Wavefront API token with permissions to query for points
+  # wavefrontToken: 12345678-90ab-cdef-0123-4567890abcde
+
+  # Metrics under this prefix are exposed in the custom metrics API.
+  metricPrefix: kubernetes
+
+  # Interval at which to fetch the list of custom metric names from Wavefront
+  metricRelistInterval: 10m
+
+  # one of info, debug, warn or trace
+  logLevel: debug
+
+  # List wavefront queries to transform into custom metrics
```

Comment:
This is actually used to drive the external metrics API (not the custom metrics). 

But additionally, this is no longer mandatory for external metrics. Starting with v0.9.3, we now support defining these via annotations on an HPA itself: https://github.com/wavefrontHQ/wavefront-kubernetes-adapter/blob/master/docs/introduction.md#annotations. So this should be optional.

Reply:
I'll leave this one to you guys if you don't mind.